pkg/model/config.go

```diff
@@ -125,14 +125,13 @@ func (c *Config) ValidateAndCompleteConfig() error {
 	// TODO(andreas): warn if user specifies tensorflow-gpu instead of tensorflow
 	// TODO(andreas): use pypi api to validate that all python versions exist
 
-	if c.Model == "" {
-		return fmt.Errorf("'model' must be set in cog.yaml")
-	}
-	if len(strings.Split(c.Model, ".py:")) != 2 {
-		return fmt.Errorf("'model' in cog.yaml must be in the form 'model.py:ModelClass")
-	}
-	if strings.Contains(c.Model, "/") {
-		return fmt.Errorf("'model' in cog.yaml cannot be in a subdirectory. It needs to be in the same directory as cog.yaml, in the form 'model.py:ModelClass")
+	if c.Model != "" {
+		if len(strings.Split(c.Model, ".py:")) != 2 {
+			return fmt.Errorf("'model' in cog.yaml must be in the form 'model.py:ModelClass")
+		}
+		if strings.Contains(c.Model, "/") {
+			return fmt.Errorf("'model' in cog.yaml cannot be in a subdirectory. It needs to be in the same directory as cog.yaml, in the form 'model.py:ModelClass")
```

Comment:
We do actually kind of support model scripts in subdirectories. If you use `workdir` the model script will be loaded from that directory.

Reply:
I didn't change this behavior -- it has shown up as an addition just because I reindented.

Reply:
Ah right, then that error message should be fixed.